package.json

```diff
@@ -10,6 +10,26 @@
   "workspaces": [
     "packages/**"
   ],
+  "release": {
+    "branches": [
+      "master",
+      "next"
```

Comment:
This repo doesn't have a `next` branch, so this can be dropped. If the `master` branch is the only branch where releases should occur, then the `branches` block can be removed since that's the default behavior.

Reply:
The `next` branch is removed. Removing branches block entirely results in error because it is required.
Error message that occurs after removing branches block: 
`A minimum of 1 and a maximum of 3 release branches are required in the branches configuration (https://github.com/semantic-release/semantic-release/blob/master/docs/usage/configuration.md#branches).`
So i think we should keep the branches block with minimum one branch.